packages/cli/commands/init.js

```diff
@@ -117,7 +119,7 @@ exports.handler = async options => {
     );
     const configPath = getConfigPath();
 
-    checkAndUpdateGitignore(configPath);
+    checkAndAddConfigToGitignore(configPath);
```

Comment:
It looks like in cli-lib the `checkAndUpdateGitignore ` function was set up to purposely fail silently in the catch block. Maybe we want to add a try/catch here so we can keep the silent fail behavior?